ubtue_Brill.js

```diff
@@ -67,7 +67,27 @@ function getSearchResults(doc) {
 	return found ? items : false;
 }
 
+// extract author names form Open Graph Metadata, if dot is included in names, e.g. "Th. Emil Homerin" https://brill.com/view/journals/jss/9/1/article-p52_3.xml 
+// because in this case the first string of the author name with dot will be removed by the system.
```

Comment:
This approach means that authors will always be extracted from Open Graph Metadata because in this one case the data `citation_author` is "Emil Homerin". Wouldn't it make more sense to inform the publisher or just live with it?

Reply:
authors will be extracted only if dot is included in the author names, that is returned by Open Graph Metadata. We could contact the publishers, but that is the only solution I have found so far.